website/docs/core-concepts/connecting-to-data-sources/authentication/README.md

```diff
@@ -1,63 +1,124 @@
 # REST APIs
 
-If your APIs are protected by an authentication mechanism, that requires a standard set of headers or parameters that need to be sent with every request, you can save these in a common data source to be reused with every request known as Authenticated APIs.
+APIs are sets of definitions and protocols that enable communication and interaction between software applications using a limited number of instructions. APIs serve as messengers, passing requests from one application to another and instantly returning a response. With Appsmith, you can connect with a wide range of tools and platforms; if there isn't a native integration with a datasource, you can connect through the REST API interface. 
```

Comment:
This part here seems redundant and may confuse the user. IMO, we should remove this part - `APIs are sets of definitions and protocols that enable communication and interaction between software applications using a limited number of instructions. APIs serve as messengers, passing requests from one application to another and instantly returning a response.`

Reply:
done